survey/src/main/java/com.quickbirdstudios.surveykit/AnswerFormat.kt

```diff
@@ -117,6 +117,34 @@ sealed class AnswerFormat {
         }
     }
 
+    object LocationAnswerFormat : AnswerFormat() {
+
+        @Parcelize
+        data class Location(val latitude: Double, val longitude: Double) : Parcelable {
+            override fun toString(): String {
+                return "${this.latitude}:${this.longitude}"
+            }
+
+            override fun equals(other: Any?): Boolean {
+                if (this === other) return true
+                if (javaClass != other?.javaClass) return false
+
+                other as Location
+
+                if (latitude != other.latitude) return false
+                if (longitude != other.longitude) return false
+
+                return true
+            }
+
+            override fun hashCode(): Int {
+                var result = latitude.hashCode()
+                result = 31 * result + longitude.hashCode()
+                return result
+            }
+        }
```

Comment:
`data classes` do not need the default `.toString()` , `.equals()` and `.hashCode()`. They are designed to generate that on their own for JVM